src/ship/activeSensors.js

```diff
@@ -37,6 +37,6 @@ export default class ActiveSensors{
                 readings.push(newReading);
             }
         }
-        return readings;
+        return new response(response=readings, success=true);
```

Comment:
If we want consistency to other `Response` objects, we might want to make the `response` an object (so it would contain a single attribute with the readings). Ultimately, it is probably ok to keep `response` as a list as long as the documentation mentions this.